specification/batch/resource-manager/Microsoft.Batch/stable/2022-10-01/BatchManagement.json

```diff
@@ -3943,7 +3947,7 @@
           "type": "string",
           "title": "Description of the private Connection state"
         },
-        "actionRequired": {
+        "actionsRequired": {
```

Comment:
This is to fix a S360 issue that was detected.  Our api was returning actionsRequired and not actionRequired.  This was broken for a while